flowmachine/flowmachine/features/subscriber/subscriber_event_count.py

```diff
@@ -0,0 +1,136 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+
+import warnings
+
+from ...core import Table
+from ..utilities.sets import EventsTablesUnion
+from .metaclasses import SubscriberFeature
+
+
+class SubscriberEventCount(SubscriberFeature):
+    """
+    This class returns the event count pr subscriber within the period,
+    optionally limited to only incoming or outgoing events.
+
+    Parameters
+    ----------
+    start, stop : str
+         iso-format start and stop datetimes
+    hours : 2-tuple of floats, default 'all'
+        Restrict the analysis to only a certain set
+        of hours within each day.
+    subscriber_identifier : {'msisdn', 'imei'}, default 'msisdn'
+        Either msisdn, or imei, the column that identifies the subscriber.
+    subscriber_subset : str, list, flowmachine.core.Query, flowmachine.core.Table, default None
+        If provided, string or list of string which are msisdn or imeis to limit
+        results to; or, a query or table which has a column with a name matching
+        subscriber_identifier (typically, msisdn), to limit results to.
+    direction : {'in', 'out', 'both'}, default 'out'
+        Whether to consider calls made, received, or both. Defaults to 'out'.
+    tables : str or list of strings, default 'all'
+        Can be a string of a single table (with the schema)
+        or a list of these. The keyword all is to select all
+        subscriber tables
+    kwargs :
+        Passed to flowmachine.EventTableSubset
+
+    Examples
+    --------
+
+    >>> s = SubscriberEventCount("2016-01-01", "2016-01-07", direction="in")
+    >>> s.get_dataframe()
+
+             subscriber  event_count
+    0  2ZdMowMXoyMByY07           65
+    1  MobnrVMDK24wPRzB           81
+    2  0Ze1l70j0LNgyY4w           57
+    3  Nnlqka1oevEMvVrm           63
+    4  4dqenN2oQZExwEK2           59
+                    ...          ...
+    """
+
+    def __init__(
+        self,
+        start,
+        stop,
+        subscriber_identifier="msisdn",
+        direction="both",
+        hours="all",
+        subscriber_subset=None,
+        tables="all",
+        **kwargs,
```

Comment:
Please no `**kwargs` any more. We used to do this pretty much everywhere in the past but have started to make all kwargs explicit (see #257), so that we can soon bundle them up and encapsulate them in more meaningful and generally smarter domain objects.

Given that `**kwargs` is only passed on to `EventsTablesUnion` and all arguments for this are already provided below, we can simply remove `**kwargs` in both places.